src/services/project-type-specifics.test.js

```diff
@@ -0,0 +1,23 @@
+import { getDocumentationLink } from './project-type-specifics';
+
+import type { ProjectType } from '../types';
+
+describe('getDocumentationLink', () => {
+  it('should get the documentation links by project type', () => {
+    const gatsbyString = getDocumentationLink('gatsby');
+    const createReactAppString = getDocumentationLink('create-react-app');
+
+    expect(typeof gatsbyString).toEqual('string');
+    expect(typeof createReactAppString).toEqual('string');
+
+    expect(gatsbyString).not.toBe(createReactAppString);
+  });
```

Comment:
Thanks for changing this :D I expect those links might change (ideally we'd have our own docs describing them, since the current docs assume CLI know-how), so it'll be nice to not have to update the tests.